lib/tasks/migrate.rake

```diff
@@ -286,6 +286,37 @@ namespace :migrate do
       end
       puts 'finished supernova src location rewrite'
     end
+
+    # Task to resolve issue where requesting a data export that has an existing export saved
+    # under the AWS path will save the newly generated export to the folder
+    # 'panoptes-uploads.zooniverse.org/{environment}/{media_type}/' rather than to
+    # the folder '{media_type}/' where it should be.
+    desc 'Update src to reflect Azure Blob Storage location for export resources'
+    task rewrite_export_src_attributes: :environment do
+      OLD_PATH_REGEX = /\A(panoptes-uploads.zooniverse.org\/#{Rails.env}\/)?(.+)\z/.freeze
```

Comment:
Ooi, I assume the prod / staging dB doesn't have any incorrect env strings, e.g. production dB has `staging` string prefix? 

If it does we might need to modify the regex to match those strings.

Reply:
Ive performed a scan of the media records on staging and prod to make sure that we don't have any private media that does not match `panoptes-uploads.zooniverse.org/{environment)` and we are good. Thanks for raising this, was a good thing to check.